README.md

```diff
@@ -32,7 +23,27 @@ Your task is to create a React element that represents _you_ (or a close approxi
 </div>
 ```
 
+To start the application type this in the terminal:
+
+```
+npm install
+npm start
+```
+
+To run the test suite type this in the terminal: 
+
+```
+npm test
+```
+
 Feel free to render out the component in the DOM using `ReactDOM.render()`.
 
+## NOTE!! 
```

Comment:
I like that idea, but.. we go over it later, and I'm not sure we want to do that kind of logic in the React.createElement() function.